sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_directory_client.py

```diff
@@ -292,7 +292,8 @@ def get_directory_properties(self, **kwargs):
         :keyword bool upn:
             Optional. Valid only when Hierarchical Namespace is
             enabled for the account. If "True", the user identity values returned
-            in the x-ms-owner, x-ms-group, and x-ms-acl response headers will be
+            in the :attr:~DirectoryProperties.owner, ~DirectoryProperties.group,
+            and DirectoryProperties.permission response headers will be
```

Comment:
Look at this in generated docs

Reply:
Trying this again with https://stackoverflow.com/questions/27170958/sphinx-documentation-how-to-reference-a-python-property

Reply:
![image](https://github.com/Azure/azure-sdk-for-python/assets/101599632/9516d373-4c87-443a-96cc-f89714ffd1ba)

Reply:
Include fully qualified name EXCLUDING attr (but we can try it)

Reply:
ALRIGHTY, renders like this: 
![image](https://github.com/Azure/azure-sdk-for-python/assets/101599632/a4a35d41-6d69-43de-8af5-7392fb7d9e4d)

I think this is the best approximation we can get here 😄 
:class:`~azure.storage.filedatalake.DirectoryProperties`